pkg/kernel/kernel.go

```diff
@@ -80,7 +80,16 @@ func (k *kernel) Add(name string, module Module) {
 		IsRunning: false,
 	}
 
-	k.modules.Store(name, state)
+	if _, didExist := k.modules.LoadOrStore(name, state); didExist {
+		// if we overwrite an existing module, the module count will be off and the application will hang while waiting
+		// until stage.moduleCount modules have booted.
+		k.logger.Panicf(
+			errors.New("module must not be redeclared"),
+			"Failed to add new module %s: module exists. Each module name needs to be unique",
```

Comment:
This may be too much nit-picking, but I noticed that most of the logs and error messages in Gosoline are written in lowercase.
So to keep consistency there I suggest doing the same.

Reply:
Also the text "Each module name needs to be unique" is redundant. We could just remove it.

Reply:
Will change it